index.js

```diff
@@ -143,13 +155,13 @@ module.exports = {
         });
       },
       linkifyTweetMentions: function (text) {
-        return text.replace(/\@[^\s\,\.\!\?\:\/]+/g, function (user) {
+        return text.replace(/@[^\W]+/g, function (user) {
```

Comment:
`^\W` is equivalent to `\w`, so that would be the best simplification of that bit. The general change here is correct though, Twitter's docs still say only A-Za-z0-9_ (i.e. `\w`) is allowed.